environments/openshift-prod-cluster.sh

```diff
@@ -45,6 +45,8 @@ export WS_K8S_API_SERVER_BASE_PATH=""
 
 export FABRIC8_SSO_API_URL=https://sso.openshift.io/
 export FABRIC8_WIT_API_URL=https://api.openshift.io/api/
+export FABRIC8_AUTH_API_URL=https://api.openshift.io/
```

Comment:
should be https://**auth**.openshift.io/**api**/, right?

Reply:
Thanks  - fixed the url too, yet to test out this script driven config.